geppetto-client/js/common/Manager.js

```diff
@@ -27,7 +27,7 @@ Manager.prototype = {
     window.Project.persisted = true;
     window.Project.readOnly = false;
 
-    GEPPETTO.trigger(GEPPETTO.Events.Project_persisted);
+    GEPPETTO.StoreManager.actionsHandler[GEPPETTO.StoreManager.clientActions.PROJECT_PERSISTED]();
```

Comment:
support for geppetto trigger is trivial and I pushed it in the current branch, having support for the old on/off methods is not rocket science but it will take me a little bit of time since not planned originally. 
Since the backward compatibility emerged just now, do we want to spend additional time in creating wrappers for GEPPETTO.on/off to mock whatever functionality was there before or do we want to put back backbone and continue to use that in parallel with redux?
The on/off mechanism will require the new eventsCallback logic to be plugged into the wrapper to take into account of the context and eventually extra variables passed to the parameter.
Right now the eventsCallback take in input only the action with eventually properties from this object that we want to have in the callback, I will have to figure out how to plug that logic in the wrapper if that is the solution that we want to follow.